### ISSUE TITLE
error[Bug] 

### ISSUE THREAD

**Issue by Dixith1999** [projectdiscovery/nuclei-templates#626]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x9bafd4]

goroutine 48174 [running]:
github.com/projectdiscovery/nuclei/v2/pkg/executer.makeHTTPClient(0x0, 0xc001ebca00, 0x0, 0x0, 0x0)
        /home/kali/nuclei/v2/pkg/executer/executer_http.go:606 +0x134
github.com/projectdiscovery/nuclei/v2/pkg/executer.NewHTTPExecuter(0xc001ebca00, 0xc001f07930, 0xc002bfc2d0, 0x895c71)
        /home/kali/nuclei/v2/pkg/executer/executer_http.go:112 +0x57
github.com/projectdiscovery/nuclei/v2/pkg/workflows.(*NucleiVar).Call(0xc002fca050, 0x0, 0x0, 0x0, 0x0, 0x0, 0x7fd7a19d27d0, 0x0)
        /home/kali/nuclei/v2/pkg/workflows/var.go:80 +0x55b
github.com/d5/tengo/v2.(*VM).run(0xc0034ea000)
        /home/kali/go/pkg/mod/github.com/d5/tengo/v2@v2.6.2/vm.go:633 +0x3323
github.com/d5/tengo/v2.(*VM).Run(0xc0034ea000, 0xce1b80, 0xc000066060)
        /home/kali/go/pkg/mod/github.com/d5/tengo/v2@v2.6.2/vm.go:77 +0xcf
github.com/d5/tengo/v2.(*Compiled).RunContext.func1(0xc001f68240, 0xc0034ea000)
        /home/kali/go/pkg/mod/github.com/d5/tengo/v2@v2.6.2/script.go:222 +0x2b
created by github.com/d5/tengo/v2.(*Compiled).RunContext
        /home/kali/go/pkg/mod/github.com/d5/tengo/v2@v2.6.2/script.go:221 +0x233

i am getting the above error while running nuclei templates.

**Comment by bauthard** [comment]
Hi @Dixith1999, 

Please follow issue guidelines while creating issues, as such can you include the required information to review this?

**Nuclei version**

```
nuclei -version 
```

**Nuclei template version**

```
cat ~/.nuclei-config.json
```

**Describe the bug**
A clear and concise description of what the bug is.

**Commands to reproduce**

```
nuclei -t xxx -target xxx
```

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Comment by bauthard** [comment]
This is fixed with the last release of nuclei v2.2.0 https://github.com/projectdiscovery/nuclei/releases
